scraper/visualize.py

```diff
@@ -91,8 +91,11 @@ def show_products(products: list[ProductInfo], title: str) -> None:
             product,
             name_format="%website - %name - %id",
         )
-    config_figure(fig, title)
-    fig.show()
+    title_ = dict(
+        text=title, font=dict(family="Courier New, monospace", color="#000000", size=18, weight="bold", style="italic")
+    )
```

Comment:
I don't like this font or formatting of the title. Can you please remove this dictionary and just pass the string variable `title` to function `config_figure`?
If the font family and other formatting settings, like font size, should be changed I would prefer it to be in the function `config_figure` together with the rest of the configuration of the figure.